CHANGELOG.md

```diff
@@ -168,6 +168,8 @@
   the Postgres connection][3593]
 - [Added `Regression` to the `Standard.Base` library and removed legacy `Model`
   type from `Standard.Table`.][3601]
+- [Added `Vector.from_polyglot_array` to make `Vector`s backed by polyglot
```

Comment:
Out of curiosity: Doesn't the Enso language provide [from conversions](https://github.com/enso-org/enso/pull/3227) - shouldn't this be a _from conversion_?

Reply:
This is an internal method and we would need an Enso type to represent the polyglot array to do it.

I think once we have this tidy up done we should follow up with the single array work suggested.